test/axelar/AxelarTransceiver.t.sol

```diff
@@ -0,0 +1,202 @@
+import {TransceiverStructs} from
+    "@wormhole-foundation/native_token_transfer/libraries/TransceiverStructs.sol";
+import {NttManager} from "@wormhole-foundation/native_token_transfer/NttManager/NttManager.sol";
+import {INttManager} from "@wormhole-foundation/native_token_transfer/interfaces/INttManager.sol";
+import {IManagerBase} from "@wormhole-foundation/native_token_transfer/interfaces/IManagerBase.sol";
+import {ERC1967Proxy} from "openzeppelin-contracts/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import {wstETHL2Token} from "../../src/token/wstETHL2Token.sol";
+
+import "forge-std/console.sol";
+import "forge-std/Test.sol";
+import "openzeppelin-contracts/contracts/access/Ownable.sol";
+
+contract AxelarTransceiverTest is Test {
+    address constant OWNER = address(1004);
+
+    uint64 constant RATE_LIMIT_DURATION = 0;
+    bool constant SKIP_RATE_LIMITING = true;
+
+    uint256 constant DEVNET_GUARDIAN_PK =
+        0xcfb12303a19cde580bb4dd771639b0d26bc68353645571a8cff516ab2ee113a0;
+
+    AxelarTransceiver transceiver;
+    IAxelarGateway gateway;
+    IAxelarGasService gasService;
+    NttManager manager;
+    wstETHL2Token token;
+
+    function setUp() public {
+        string memory url = "https://ethereum-sepolia-rpc.publicnode.com";
+        vm.createSelectFork(url);
+
+        gateway = IAxelarGateway(new MockAxelarGateway());
+        gasService = IAxelarGasService(address(new MockAxelarGasService()));
+
+        token = new wstETHL2Token("name", "symobl", OWNER, OWNER);
+        address managerImplementation = address(
+            new NttManager(
+                address(token),
+                IManagerBase.Mode.LOCKING,
+                1,
+                RATE_LIMIT_DURATION,
+                SKIP_RATE_LIMITING
+            )
+        );
+        manager = NttManager(address(new ERC1967Proxy(managerImplementation, "")));
+        manager.initialize();
+        manager.transferOwnership(OWNER);
+        address implementation =
+            address(new AxelarTransceiver(address(gateway), address(gasService), address(manager)));
+        transceiver = AxelarTransceiver(address(new ERC1967Proxy(implementation, "")));
+        transceiver.initialize();
+        vm.prank(OWNER);
+        manager.setTransceiver(address(transceiver));
+    }
+
+    function test_setAxelarChainId() public {
+        uint16 chainId = 1;
+        string memory chainName = "chainName";
+        string memory axelarAddress = "axelarAddress";
+
+        vm.prank(OWNER);
+        transceiver.setAxelarChainId(chainId, chainName, axelarAddress);
+        /*assertEq(transceiver.idToAxelarChainIds(chainId), chainName);
+        assertEq(transceiver.axelarChainIdToId(chainName),chainId);
+        assertEq(transceiver.idToAxelarAddress(chainId), axelarAddress);
+        assertEq(transceiver.axelarAddressToId(axelarAddress), chainId);*/
+    }
+
+    function testFail_setAxelarChainIdNotOwner() public {
```

Comment:
Can you change all the `testFail` tests to `test` and then use `expectRevert` with a selector instead please?

`testFail` succeeds on any revert, but ideally we want to expect a specific revert error message/selector in the tests